### PR TITLE
More typo and wording changes

### PR DESCRIPTION
**Pull request by CoffeeBrewer64**
Changed some more text
Fixed more typos
Added logging about controller port being empty, switching to keyboard input and one more (ctrl.cpp)

### CODE REVIEW

**Comment by CoffeeBrewer64** [comment]
Finalised, waiting for review.

**Comment by CoffeeBrewer64** [comment]
@sunho review? 🥺🥺

File: vita3k/gui/src/settings_dialog.cpp [hunk 1]
```diff
@@ -650,7 +650,7 @@ void draw_settings_dialog(GuiState &gui, EmuEnvState &emuenv) {
         ImGui::Spacing();
         ImGui::TextColored(GUI_COLOR_TEXT, "Enter button assignment \nSelect your 'Enter' button.");
         if (ImGui::IsItemHovered())
-            ImGui::SetTooltip("This is the button that is used as 'Confirm' in applications dialogs. \nSome applications don't use this and get default confirmation button.");
+            ImGui::SetTooltip("This is the button that is used as 'Confirm' in applications' dialogs. \nSome applications don't use this and use the default confirmation button.");
```
Comment (EXtremeExploit) [inline vita3k/gui/src/settings_dialog.cpp:653]: forgor to delete extra ' here

**Review by EXtremeExploit — CHANGES_REQUESTED** [review]

File: vita3k/gui/src/settings_dialog.cpp (hunk 1 above)
Comment (CoffeeBrewer64) [reply vita3k/gui/src/settings_dialog.cpp:653]: it's a possessive apostrophe for multiple things so it needs a '
🤓

**Comment by Zangetsu38** [comment]
no need add \n for log, and 21 commit, please squash

File: vita3k/gui/src/settings_dialog.cpp (hunk 1 above)
Comment (CoffeeBrewer64) [reply vita3k/gui/src/settings_dialog.cpp:653]: fixed

File: README.md [hunk 2]
```diff
@@ -32,10 +33,31 @@ The emulator currently runs some homebrew programs. It is also able to load some
...
+Recommended:
+
+* GPU that supports OpenGL 4.4
+* GPU that supports shader interlock
+* CPU with the AVX instruction set
+* 8GB of RAM or greater
+
+Your device **must** have a 64 bit operating system for Vita3K to run.
+
+(Windows) If you get a "VCRUNTME140_1.dll was not found" error, download and install the [Visual C++ 2015-2019 Redistributable](https://aka.ms/vs/16/release/vc_redist.x64.exe).
```
Comment (Zangetsu38) [inline README.md:57]: Is wrong, is runtime for vs2022 here

File: vita3k/ctrl/src/ctrl.cpp [hunk 3]
```diff
@@ -176,6 +180,7 @@ static void apply_keyboard(uint32_t *buttons, float axes[4], bool ext, EmuEnvSta
     axes[1] += keys_to_axis(keys, static_cast<SDL_Scancode>(emuenv.cfg.keyboard_leftstick_up), static_cast<SDL_Scancode>(emuenv.cfg.keyboard_leftstick_down));
     axes[2] += keys_to_axis(keys, static_cast<SDL_Scancode>(emuenv.cfg.keyboard_rightstick_left), static_cast<SDL_Scancode>(emuenv.cfg.keyboard_rightstick_right));
     axes[3] += keys_to_axis(keys, static_cast<SDL_Scancode>(emuenv.cfg.keyboard_rightstick_up), static_cast<SDL_Scancode>(emuenv.cfg.keyboard_rightstick_down));
+    LOG_INFO("Using keyboard instead of controller.");
```
Comment (Zangetsu38) [inline vita3k/ctrl/src/ctrl.cpp:183]: I no see any utility of this, is juste spam log for nothing, same’for other log in this file

File: README.md [hunk 4]
```diff
@@ -12,10 +12,11 @@ Vita3K is an experimental PlayStation Vita emulator for Windows, Linux and macOS
 * [**Discord**](https://discord.gg/MaWhJVH) (recommended)
 * IRC `#vita3k` on **freenode** ([Web-based IRC client](https://webchat.freenode.net/?channels=%23vita3k))
 * [Patreon](https://www.patreon.com/Vita3K) (support the project)
+* [Blog](https://vita3k.org/blog.html)
```
Comment (nishinji) [inline README.md:15]: I don't think it's necessary.

**Pull request closed without merging**